docs/docs/run-node/01-run-node.md

```diff
@@ -63,6 +63,13 @@ jq '.app_state.gov.voting_params.voting_period = "600s"' genesis.json > temp.jso
 jq '.app_state.mint.minter.inflation = "0.300000000000000000"' genesis.json > temp.json && mv temp.json genesis.json
 ```
 
+### Client Interaction
+
+When instantiating a node Grpc and REST are deafaulted to localhost to avoid unknown exposure of your node to the public. It is recommended to not expose these endpoints without a proxy that can handle loadbalancing or authentication is setup between your node and the public. 
```

Comment:
```suggestion
When instantiating a node, GRPC and REST are defaulted to localhost to avoid unknown exposure of your node to the public. It is recommended to not expose these endpoints without a proxy that can handle load balancing or authentication is set up between your node and the public. 
```